src/OrbitGl/Batcher.cpp

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2023 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitGl/Batcher.h"
+
+namespace orbit_gl {
+bool operator==(const Batcher::Statistics& lhs, const Batcher::Statistics& rhs) {
```

Comment:
Why don't you implement that in the struct/class itself?

Reply:
Hm I thought we had the rule to declare unary operators within the class, and binary / tertiary outside in the style guide, but I was wrong. Anyways that is usually how I treat it, it's more consistent if the arguments can be cast, or are of different types and you'd want to provide both orders of parameters.